docker/dev/centos/README.md

```diff
@@ -0,0 +1,46 @@
+# How to use this Dockerfile
```

Comment:
I agree in having the .md documentation along with the container directory in the repo, but this should be cited/linked in the README.md file in the repository root.